### PR TITLE
correction for the hw

### PR DESCRIPTION
**Pull request by ramla199**
# Your Details

- Your Name: Ramla Jama
- Your City: london
- Your Slack Name: Ramla 

# Homework Details

- Module: git and github
- Week: 1

### CODE REVIEW

**Comment by AbdulazizAlhasan** [comment]
Good work Ramla.

File: homework.md [hunk 1]
```diff
@@ -44,70 +44,77 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used on the website to make intractive website
```
Comment (oksygenn) [inline homework.md:47]:
```suggestion
A programming language used on the website to make interactive website
```
a small typo :)

File: homework.md [hunk 2]
```diff
@@ -44,70 +44,77 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-A place for us to store and share our code.
+is a code hosting platform for version control and collaboration.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+save your changes
```
Comment (oksygenn) [inline homework.md:102]: Hi Ralma, could you please elaborate where does it save changes?

**Review by oksygenn — COMMENTED** [review]
Hi Ralma. 
Great answers! Please find a few improvement suggestions below. Also it would be great to change your pull request's title to something like: London Class 7 - your name - Week 1.
 Thanks

File: homework.md [hunk 3]
```diff
@@ -4,38 +4,38 @@
...
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+styling a website to look good
```
Comment (ademalagoz) [inline homework.md:31]: Hi Ramla. Great job. 
Can you give more details please like colour, layout etc...

File: homework.md [hunk 4]
```diff
@@ -44,70 +44,77 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-To record what changes you made to the code
+To record a particular  changes you made to the code
 
 ## Question 18
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+The Git Cheatsheet and by asking on slack!
```
Comment (ademalagoz) [inline homework.md:120]: Good answer!

**Comment by ademalagoz** [comment]
One more thing. As teachers said you should change your title like below.
London Class 7 - Adem Alagoz - Git&GitHub - Week 1

**Pull request closed without merging**